src/screens/WalletConnectApprovalSheet.js

```diff
@@ -223,11 +223,13 @@ export default function WalletConnectApprovalSheet() {
   const handlePressChangeWallet = useCallback(() => {
     Navigation.handleAction(Routes.CHANGE_WALLET_SHEET, {
       currentAccountAddress: approvalAccount.address,
-      onChangeWallet: (address, wallet) =>
-        setApprovalAccount({ address, wallet }),
+      onChangeWallet: (address, wallet) => {
+        goBack();
+        setApprovalAccount({ address, wallet });
```

Comment:
i feel like it would be better for these lines to be flipped in order, goBack after setAccount

Reply:
yeah I thought the same but we're not going back from this component, we're just closing the sheet on top of it. I can do it tho

Reply:
done 👌

Reply:
thank u ser, merge whenever 🙏